### DISCUSSION TITLE
Can no longer ask Siri to "open" or "close" my HomeSpan controlled blinds

### DISCUSSION THREAD

**Discussion by dburr** [HomeSpan/HomeSpan#290] [Q&A] [answered]
First, thanks for putting out such a FANTASTIC library!

Unfortunately I've run into some trouble today...

I've been happily using HomeSpan to control my office blinds. If I asked Siri to "open the blinds" or "close the blinds" she would do it without hesitation. Until today, that is. Now when I ask her to open or close the blinds, she comes back with "there are no accessories that support that." Yet my Blinds still show up in the Home app, and if I tap on them in the app, they still open and close like normal.

The only thing I can think of that changed recently is that this week I upgraded to the latest versions of iOS (15.4) and watchOS (8.5)

Have tried upgrading HomeSpan to 1.5.0 (was running 1.3.0 previously) but unfortunately that did not fix the issue either.

For now I've worked around the problem by creating scenes for blinds open ("Natural light") and blinds closed ("artificial light")  But I'm curious as to what changed that caused my previously-working "open/close the blinds" commands to stop working? Any ideas?

**Comment by HomeSpan** [comment]
This does not seem to be a HomeSpan issue but rather a Siri issue that is unfortunately not that uncommon.  Every time Apple updates Siri there seem to be subtle changes in how it processes voice commands.  I have noticed in the past that sometimes things simply start working again as Apple fixes these issues behind the scenes.  Sometimes the problem is that one of my HomePods is still running the prior iOS version (in which case I usually need to cycle power a few times to get it to trigger an update). In other cases Siri responds correctly when I speak into the iPhone, but differently when a HomePod picks up my voice instead.  

You can also try some other phrases, like "turn on the blinds," which may trigger the appropriate reaction from Siri.

**Comment by HomeSpan** [comment, marked as answer]
Looks like this issue is widespread:  https://www.reddit.com/r/HomeKit/comments/te7s4u/ios_154_siri_no_longer_understands_how_to_operate/
